docs/release-notes/release-notes-0.18.0.md

```diff
@@ -286,6 +286,9 @@ bitcoin peers' feefilter values into account](https://github.com/lightningnetwor
   callbacks](https://github.com/lightningnetwork/lnd/pull/8504) which will
   execute whenever a healthcheck succeeds/fails.
 
+* [Remove io/ioutil package
```

Comment:
Sorry, one last request: I think this would be better suited for the `## Code Health` section further below.

Reply:
@guggero No problem. There you go. I moved the note to the ## Code Health section.